### PR TITLE
Removes transnational values from worksheet 28b, 28c regional reports

### PR DESCRIPTION
**Pull request by esirK**
## Description

Removes transnational values from worksheet 28b, 28c regional reports

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
Screenshot 2021-04-09 at 10 34 13

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: reports/report_builder.py [hunk 1]
```diff
@@ -1240,7 +1240,7 @@ def ws_28b(self, ws):
         c = 1
         r = 8
         write_row_headings = True
-        all_regions = add_transnational_to_regions(self.regions)
+        all_regions = add_transnational_to_regions(self.regions) if self.report_type == 'global' else self.regions
```
Comment (kilemensi) [inline reports/report_builder.py]: This is needed in [ws_28](https://github.com/CodeForAfrica/gmmp/pull/175/files#diff-19a686dbe88bb9f2f8c4f2ce5e331aa6f6db260926a2a5b7f1edfa7864a649b4L1186) as well.

**Review by kilemensi — COMMENTED** [review]
👍🏾 @esirK 

------------------------

I don't think [`self.region_list`](https://github.com/CodeForAfrica/gmmp/pull/175/files#diff-19a686dbe88bb9f2f8c4f2ce5e331aa6f6db260926a2a5b7f1edfa7864a649b4L1289) includes `Transnational`. We may have to create `all_region_list` from `all_regions`.

**Comment by esirK** [comment]
@kilemensi  
something like 
```
all_region_list = add_transnational_to_regions(self.regions)
self.region_list = [name for id, name in all_region_list]
```
??

**Comment by kilemensi** [comment]
> @kilemensi
> something like
> 
> ```
> all_region_list = add_transnational_to_regions(self.regions)
> self.region_list = [name for id, name in all_region_list]
> ```
> 
> ??

Yes @esirK but lets not **override** `self.region_list`.

**Comment by esirK** [comment]
> 
> Yes @esirK but lets not **override** `self.region_list`.

Not sure I 100% follow. Don't we want `self.region_list` to always have `Transnational`?

**Comment by kilemensi** [comment]
> > Yes @esirK but lets not **override** `self.region_list`.
> 
> Not sure I 100% follow. Don't we want `self.region_list` to always have `Transnational`?

For queries that uses Transnational, yes we want it. But for queries that don't include Transnational, we shouldn't force it there @esirK .

**Comment by kilemensi** [comment]
> > > Yes @esirK but lets not **override** `self.region_list`.
> > 
> > 
> > Not sure I 100% follow. Don't we want `self.region_list` to always have `Transnational`?
> 
E.g. query 30 doesn't use Transnational (unless it has been updated). This means if we update `self.region_list`, `ws_30` will also now include Transnational.

**Comment by esirK** [comment]
Ohh. I see. I've made the updates.

File: reports/report_builder.py [hunk 2]
```diff
@@ -149,6 +149,8 @@ def __init__(self, form):
             self.report_type = 'global'
 
         self.country_list = [code for code, name in self.countries]
+        all_regions = add_transnational_to_regions(self.regions)
+        self.all_region_list = [name for id, name in all_regions]
```
Comment (kilemensi) [inline reports/report_builder.py]: We shouldn't have this on `self`, just like we don't have `all_regions` on `self`... it's query dependent.

**Review by kilemensi — APPROVED** [review]
🚀 @esirK

**Pull request merged**